### PR TITLE
chore: ethers v6

### PR DESCRIPTION
**Pull request by iankressin**

### CODE REVIEW

File: src/Verify/Etherscan.ts [hunk 1]
```diff
@@ -140,7 +140,7 @@ export class Etherscan {
             )
         }
 
-        return await response.body.json()
+        return await response.body.json() as EtherscanResponseBody
```
Comment (iankressin) [inline src/Verify/Etherscan.ts:143]: `response.body.json` was `unknow`, but that has nothing to do with the newer versions of the packages, I believe

File: src/helpers/artifacts.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import ERC1967Proxy from '@openzeppelin/upgrades-core/artifacts/@openzeppelin/co
 import UpgradesBuildInfo from '@openzeppelin/upgrades-core/artifacts/build-info.json'
 
 import VanityDeployer from '../../artifacts/contracts/VanityDeployer.sol/VanityDeployer.json'
-import VanityDeployerBuildInfo from '../../artifacts/build-info/2784e0ae65b7a935c863195be5ba6120.json'
+import VanityDeployerBuildInfo from '../../artifacts/build-info/5938466a8d937f61d995799b999eaeb1.json'
```
Comment (iankressin) [inline src/helpers/artifacts.ts:5]: the old hash changed because of the changes in the `VanityDeployer` contract

File: tsconfig.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "target": "ES2018",
+        "target": "ES2020",
```
Comment (iankressin) [inline tsconfig.json:3]: We're unable to use `bigint` with `ES2018`

File: contracts/VanityDeployer.sol [hunk 4]
```diff
@@ -47,7 +47,7 @@ contract VanityDeployer is OwnableUpgradeable, UUPSUpgradeable {
         emit DeployedContract(addr, true);
     }
 
-    function getAddress(
+    function calculateAddress(
```
[the remark this answers is not in the document]
Comment (RauppRafael) [reply contracts/VanityDeployer.sol:50]: lulz

**Review by RauppRafael — APPROVED** [review]

**Pull request merged**